tests/data/expected/main/openapi/cyclic_component_refs.py

```
@@ -0,0 +1,19 @@
+#   timestamp: 2019-07-26T00:00:00+00:00
+
+from __future__ import annotations
+
+from pydantic import BaseModel
+
+
+class Model(BaseModel):
+    pass
+
+
+class Person(Team, Model):
```

Ruff (0.15.9)

[error] Undefined name `Team`

(F821)